### ISSUE TITLE
Add support for non-namespaced params

### ISSUE THREAD

**Issue by vlucas** [heartcombo/simple_form#821]
I am trying to use simple_form to create a form that will do a simple GET request to the current page for a global context switch functionality. The form will be on every page, and can be submitted to whatever page that is (dynamic and unknown when form is rendered). I am having an incredibly difficult time doing this with simple_form.

Current code:

```
= simple_form_for '', method: :get do |f|
  = f.input :year, collection: (Time.now.year).downto(2011).to_a, selected: (params[:year] || Time.now.year)
  = f.input :quarter, collection: 1..4, selected: (params[:quarter] || (((Time.now.month - 1) / 3) + 1))
  = f.submit 'Set Quarter'
```

This currently produces a query string like this:
`[year]=2013&[quarter]=4`

With an empty string, the expected query string output would be:
`year=2013&quarter=4`

In other words, without any namespacing of the params with brackets. This could be fixed by either allowing an empty string to produce bracket-less params as a special case, or adding an additional option flag that would turn off the param namespacing.
